src/js/entities-service/actions.js

```diff
@@ -27,6 +33,30 @@ const Entity = BaseEntity.extend({
 
     return this.fetchCollection({ url });
   },
+  fetchActionsByDate({ filter }) {
+    const data = { filter };
+
+    return this.fetchCollection({ data })
+      .then(response => {
+        const groupedCollection = response.groupBy('due_date');
+        const arrayByDate = reduce(keys(groupedCollection), (collection, key) => {
+          return collection.concat({
+            date: key,
+            actions: new Backbone.Collection(groupedCollection[key], {
```

Comment:
I don't think we want this to be a generic backbone collection.  Seems like it should be `Collection` in this context would be an actions collection.

Also it might be worth passing the `new Backbone.Collection` as the memo instead of `[]` to the reduce such that instead of array.concat it'd be `collection.add` then `return collection`

Also, rethinking this, let's sort both on the collectionviews and not in the data.